go/vt/dbconfigs/dbconfigs.go

```diff
@@ -135,69 +135,83 @@ func registerPerUserFlags(dbc *userConfig, userKey string) {
 
 }
 
+// ConnWrapper contains mysql.connparams for now
+type ConnWrapper struct {
+	connWrapperParams *mysql.ConnParams
+}
+
+// GetConnParams returns the connections params
+func (cw ConnWrapper) GetConnParams() *mysql.ConnParams {
```

Comment:
Rename to `ConnParams`.
This should invoke `WithCredentials`, and we should delete all call sites that use `WithCredentials`.

This also means that the function can return an error, and you'll have to check for it at all call sites.

Reply:
Done.